test/checks/color/color-contrast.js

```diff
@@ -157,4 +157,39 @@ describe('color-contrast', function () {
 		assert.equal(checkContext._data.missingData[0].reason, 'bgGradient');
 		assert.equal(checkContext._data.contrastRatio, 0);
 	});
+
+	it('should return undefined when there are elements overlapping', function () {
```

Comment:
I'm not sure I understand why this case needs to be reviewed. The text is completely obscured, so there is no contrast issue, right?